h/static/styles/partials-v2/_base.scss

```diff
@@ -52,6 +52,11 @@ $max-phone-width: 500px;
 $tablet-width: 768px;
 $touch-target-size: 44px;
 
+// Misc
+// ----
+$search-result-container-margin-left: 30px;
+$search-result-container-margin-right: 60px;
```

Comment:
Wasn't sure where to put these variables, since they need to be accessed by two other files, there didn't seem to be any place other than `_base.scss`